internal/database/repository/node_int_test.go

```diff
@@ -22,6 +22,8 @@ type NodeTestSuite struct {
 	repo *PostgresRepository
 }
 
+var partitionID = ulid.Make().String()
```

Comment:
```suggestion
partitionID := ulid.Make().String()
```
looks better I guess